internal/server/data/group_test.go

```diff
@@ -126,6 +126,41 @@ func TestListGroups(t *testing.T) {
 			}
 			assert.DeepEqual(t, actual, expected, cmpGroupShallow)
 		})
+
+		t.Run("filter identities by group", func(t *testing.T) {
```

Comment:
This should be in `data/identity_test.go`

Reply:
moved.